enabler/src/components/payment-methods/klarnaPayLater.ts

```diff
@@ -0,0 +1,56 @@
+import Core from "@adyen/adyen-web/dist/types/core/core";
```

Comment:
judging by the other files changed in this PR, should we rename these files to be kebab case? 
e.g. `klarnaPayLater.ts` -> `klarna-pay-later.ts`